components/src/Button/Button.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import styled, { css } from 'styled-components';
 import { colour, space, font, radius, shadow } from '@nulogy/tokens';
+import { Icons } from '../Icons';
+const {Icon} = Icons;
```

Comment:
These imports are not used. remove them.

Reply:
actually I'm in this PR now, making the change to wrap the children in a span for styling